Remotion/Web/Core/UI/Controls/CommandInfo.cs

```diff
@@ -157,12 +157,27 @@ private void AddDiagnosticMetadataAttributes (HtmlTextWriter writer)
 
     private bool IsTriggeringNavigation ()
     {
-      return _href != "#";
+      if (_href == null)
+        return false;
+      else if (_href.StartsWith("#"))
+        return false;
+      else if (_onClick == null)
+        return true;
+      else if (_onClick.Contains("__doPostBack"))
+        return false;
+      else
+        return true;
     }
 
     private bool IsTriggeringPostBack ()
     {
-      return !IsTriggeringNavigation() && _onClick!.Contains("__doPostBack"); // TODO RM-8104: Guard _onClick
+      if (IsTriggeringNavigation())
+        return false;
+      else if (_onClick!.Contains("__doPostBack"))
+        return true;
+      else
+        return false;
```

Comment:
```suggestion
      if (_onClick == null)
        return false;
      else if (_onClick.Contains("__doPostBack"))
        return true;
      else
        return false;
```

We no longer need the test for IsTriggeringNavigation because onclick starting a postback implies there's no navigation possible.